nimare/meta/cbma/mkda.py

```diff
@@ -476,26 +481,26 @@ def _fit(self, dataset1, dataset2):
 
         del n_mappables
 
-        # Conditional probabilities
         pAgF = n_selected_active_voxels / n_selected
         pAgU = n_unselected_active_voxels / n_unselected
         pFgA = pAgF * pF / pA
 
         del pF
 
-        # Recompute conditionals with uniform prior
-        pAgF_prior = self.prior * pAgF + (1 - self.prior) * pAgU
-        pFgA_prior = pAgF * self.prior / pAgF_prior
+        if self.prior is not None:
```

Comment:
```suggestion
        if self.prior:
```
oh I guess the if statement is there twice